crates/core/tedge/src/cli/certificate/renew.rs

```diff
@@ -59,6 +59,8 @@ mod tests {
     use std::path::Path;
```

Comment:
Wondering if it'd be safer to call `set_device_id` from this `renew` command `execute()` as well so that any configuration from an older version of `tedge` is updated when the cert is renewed with the newer version.

Reply:
Why not :+1:

Reply:
Done in 56986f249540b7872f2b8807663fd77b6e6266ad